cluster-sim/cluster_sim/locale/ru/LC_MESSAGE/cluster_sim.po

```diff
@@ -0,0 +1,966 @@
+"%s\r\n"
+"\n"
+"     <ESC>- Back, <ENTER>- Refresh, <CTRL-L>- Event Log\n"
+"          - Назад        - Обновить         - Протокол\n"
+"> "
+
+#: fake\_power\_control.py:326
+msgid ""
+"\n"
+"------- Outlet %s -------------------------------------------------------------\n"
+"\n"
+"        Name         : Outlet %s\n"
+"        Outlet       : %s\n"
+"        State        : %s\n"
+"\n"
+"     1- Control Outlet\n"
+"     2- Configure Outlet\n"
+"\n"
+"     ?- Help, <ESC>- Back, <ENTER>- Refresh, <CTRL-L>- Event Log\n"
+"> "
+msgstr ""
+"\n"
+"------- Outlet %s -- Розетка --------------------------------------------------\n"
+"\n"
+"        Name   Имя       : Outlet %s\n"
+"        Outlet Розетка   : %s\n"
+"        State  Состояние : %s\n"
+"\n"
+"     1- Control Outlet   -- Управлять розетку\n"
+"     2- Configure Outlet -- Конфигурировать розетку\n"
+"\n"
+"     ?- Help, <ESC>- Back, <ENTER>- Refresh, <CTRL-L>- Event Log\n"
+"      - Помощь     - Назад        - Обновить         - Протокол\n"
+"> "
+
+#: fake\_power\_control.py:344
+msgid ""
+"\n"
+"------- Control Outlet --------------------------------------------------------\n"
+"\n"
+"        Name         : Outlet %s\n"
+"        Outlet       : %s\n"
+"        State        : %s\n"
+"\n"
+"     1- Immediate On\n"
+"     2- Immediate Off\n"
+"     3- Immediate Reboot\n"
+"     4- Delayed On\n"
+"     5- Delayed Off\n"
+"     6- Delayed Reboot\n"
+"     7- Cancel\n"
+"\n"
+"     ?- Help, <ESC>- Back, <ENTER>- Refresh, <CTRL-L>- Event Log\n"
+"> "
+msgstr ""
+"\n"
+"------- Control Outlet -- Управлять розеткой ----------------------------------\n"
+"\n"
+"        Name   Имя       : Outlet %s\n"
+"        Outlet Розетка   : %s\n"
+"        State  Состояние : %s\n"
+"\n"
+"     1- Immediate On     -- Немедленное включение\n"
+"     2- Immediate Off    -- Немедленное отключение\n"
+"     3- Immediate Reboot -- Немедленная перезагрузка\n"
+"     4- Delayed On       -- Отложенное включение\n"
+"     5- Delayed Off      -- Отложенное отключение\n"
+"     6- Delayed Reboot   -- Отложенная перезагрузка\n"
+"     7- Cancel           -- Отмена\n"
+"\n"
+"     ?- Help, <ESC>- Back, <ENTER>- Refresh, <CTRL-L>- Event Log\n"
+"      - Помощь     - Назад        - Обновить         - Протокол\n"
+"> "
+
+#: fake\_power\_control.py:367
+msgid ""
+"\n"
+"        -----------------------------------------------------------------------\n"
+"        I'm too lazy to go through all the various permutations.\n"
+"        The key line expected by fence\_apc is the next one:\n"
+"\n"
+"        Enter 'YES' to continue or <ENTER> to cancel : "
+msgstr ""
+"\n"
+"        -----------------------------------------------------------------------\n"
+"        Я слишком ленив, чтобы продираться через всевозможные перестановки.\n"
+"        Нужная строчка, которую ждёт fence\_apc, вот эта:\n"
+"\n"
+"        Введите 'YES', чтобы продолжить или <ENTER>, чтобы отменить: "
```

Comment:
How do we know that all of the above translations are accurate to the translations on the actual PDU being simulated.

Reply:
The only way to ensure it is to find the relevant APC manual for respective market and grab the text from there... Would it be a copyright violation, BTW?

Or it may be left intact (no translation).
Or provide it "as is".
Or provide a mean to select "enforced translation" at command line or config...

Reply:
BTW, I ain't sure, if it's a real device message... It sounds way too jokingly to be hardwired into equipment...

Reply:
Or we just don't translate the simulatd PDU interface and require that (until such questions can be answered, and translations updated accurately) IML users set their PDU interface to be default English locale.

The simulated PDU interface shouldn't be translated until the IML code that interfaces with it has translations is localized in any case.

Reply:
ok